spec/views/shared/_one_time_code_input.html.erb_spec.rb

```diff
@@ -98,4 +98,25 @@
       expect(rendered).to have_selector('[data-transport][data-foo="bar"]')
     end
   end
+
+  describe 'maxlength' do
+    context 'no maxlength given' do
+      it 'renders input maxlength DIRECT_OTP_LENGTH' do
+        puts rendered
```

Comment:
🙃 

```suggestion
```

Reply:
Whoops, fix is here: https://github.com/18F/identity-idp/pull/6672